src/lv_interop.cc

```diff
@@ -9,17 +9,22 @@
 #include <dlfcn.h>
 #endif
 
+static int kCleanOnRemove = 0;
```

Comment:
lets add some brief comments here to explain there purpose

Reply:
Done.